Software/app/basic_bootloader/README.md

```diff
@@ -0,0 +1,89 @@
+# basic_bootloader
+
+## Description
+
+The application provides a basic bootloader that can be combined with other applications to perform a specific functionality at boot time.
+
+> If you want to learn more about the purpose of a bootloader, we recommend reading this great [blog post](chttps://interrupt.memfault.com/blog/how-to-write-a-bootloader-from-scratch).
```

Comment:
This does not seem to produce a clickable link for me, probably because of the stray 'c' before https (but double-check because I would expect it to take me to a non-existent website or something, but I can't interact at all which is weird).

Reply:
good catch